hw1/02-palindrome.html

```diff
@@ -1,27 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>02 - Palindrome</title>
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
-    <script defer src="02-palindrome.js"></script>
-  </head>
-  <body>
-    <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
-        </div>
-        <div>
-          <p>Is this a palindrome?</p>
-          <div></div>
-        </div>
-      </section>
-    </main>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>02 - Palindrome</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+  <script defer src="02-palindrome.js"></script>
+</head>
+
+<body class="vh-100">
+  <main class="bg-primary h-100 pt-4">
+    <section class="card mx-auto w-50 p-3">
+      <h1>Exercise 02 - Palindrome</h1>
+
+      <div class="mt-4">
+        <label>Enter a positive number:</label>
+        <input type="number" />
```

Comment:
This input tag will generate an error on Wave saying "Form control does not have a corresponding label". To fix both the label and input error on Wave add the "id" attribute in the input tag and the "for" attribute in the label tag.

Reply:
I don't think using `id` is a good idea in general for forms like this, I'll use `name` and `for` instead. Thanks for the suggestion!

Reply:
You are welcome! Just make sure you check for errors and alerts on the Wave before turning your assignment in.